test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -266,4 +285,90 @@ public void restartTest() throws Exception {
         assertThat(corfuRuntime.getLayoutView().getLayout().getEpoch()).isEqualTo(l.getEpoch() + 1);
         assertThat(shutdownCorfuServer(corfuServer)).isTrue();
     }
+
```

Comment:
When running our kill node test, with this PR on top of latest master client hangs and test is never able to terminate (regardless of the node being killed, i.e., log unit with primary sequencer or different node).

Reply:
Resolved. The test framework was hanging as there was a change in the log directory.